oar/core/config_store.py

```diff
@@ -0,0 +1,135 @@
+import requests
+import json
+import yaml
+import os
+import oar.core.util as util
+from oar.core.exceptions import ConfigStoreException
+from requests.exceptions import RequestException
+from yaml import YAMLError
+
+
+class ConfigStore:
+    """
+    Config Store object will be used in other modules to retrieve
+    release data and app level config settings. e.g. advisories, builds etc.
+
+    cs = ConfigStore("4.12.11")
+    ads = cs.get_advisories()
+
+    """
+
+    def __init__(self, release):
+        if release == "":
+            raise ConfigStoreException("argument release is required")
```

Comment:
@rioliu-rh maybe we could use Decorator
https://github.com/miguelgrinberg/merry
https://code.tutsplus.com/tutorials/professional-error-handling-with-python--cms-25950

we could consider it late.

Reply:
ok.